app/javascript/spec/main-menu/__snapshots__/main-menu.spec.js.snap

```diff
@@ -191,7 +191,7 @@ exports[`Main menu test should render correctly 1`] = `
             >
               <li
                 className="active list-group-item secondary-nav-item-pf"
-                data-target="#menu-vi"
+                data-target="menu-vi"
```

Comment:
This is wrong... https://getbootstrap.com/docs/3.3/javascript/

All the examples in bootstrap docs use data-target with the `#`.

Or is there any patternfly code treating data-target differently?

Reply:
@himdel that is a typo.

Reply:
Sorry, # is back in targets now :)